### ISSUE TITLE
Upgrade to use code from wxPython 4

### ISSUE THREAD

**Issue by josephsl** [josephsl/stationPlaylist#31]
Hi,

As noted in NVDA Core issue 7077: wxPython 4 introduces changes such as no more wx.PyDeadObjectError and many things. In case of this add-on, a concern is lack of wx.CENTER_ON_SCREEN. Because it is risky to introduce this into stable builds, it'll be incubated for a long time and should be prepared for release in summer.

Thanks.

**Comment by josephsl** [comment]
Hi,

Fixed in 17.08 stable, partially backported to 15.x LTS. Thanks.
